specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/applicationGateway.json

```diff
@@ -1895,6 +1920,28 @@
       ],
       "description": "Allows to disable rules within a rule group or an entire rule group."
     },
+    "ApplicationGatewayFirewallExclusion": {
+      "properties": {
```

Comment:
when i tested i noticed it was like this
   "exclusions": [
            	{
        			"matchVariable": "RequestCookieNames",
        			"selectorMatchOperator": "Contains",
        			"selector": "hello"
            	}
        	] 
There was no "properties" : {}. So is this correct?